### DISCUSSION TITLE
How to remove "GitHub repository" / "Keystone JS" documentation link in Admin UI

### DISCUSSION THREAD

**Discussion by Adityacs001** [keystonejs/keystone#7555] [Questions] [answered]
Please guide with step to customize loggedin menu pop to edit logged in user menu view.

Also Can we customize SignIn Page to have background Image.

**Comment by dcousens** [comment, marked as answer]
Hi @Adityacs001,

These links have been removed when `NODE_ENV==='production'` in https://github.com/keystonejs/keystone/pull/7546,  please let me know if that doesn't answer your question.

**Reply by Adityacs001** [reply]
Hi @dcousens thanks for your prompt response. Yes its working .Addition to environment variable ,  I had to update keystone packages.

**Reply by dcousens** [reply]
The next release should be published in the next few days
